public/vislib/vector_layer_types/EsLayer.js

```diff
@@ -130,7 +138,11 @@ export default class EsLayer {
           }
         );
         self.bindPopup(layer, options);
-        layer.icon = `<i class="far fa-stop" style="color:${layerControlColor};"></i>`;
+        if (geo.type.includes('line')) {
```

Comment:
Can we use the syntax as below, just in case there is something else that may contain line: 


```suggestion
        if (geo.type === 'linestring' || geo.type === 'multilinestring) {
```